plugins/mumble_plugin_win32.h

```diff
@@ -18,9 +18,24 @@
 
 #include "mumble_plugin.h"
 
+#if _MSC_VER < 1800 && defined(M_IX86)
+typedef unsigned long long uintptr64_t;
+typedef unsigned long uintptr32_t;
+#else
+#include <stdint.h>
+typedef uint64_t uintptr64_t;
```

Comment:
Do you like procptr32_t and procptr64_t? I like them because they go well with the name peekProc.